plugins/woocommerce-blocks/assets/js/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -46,6 +46,7 @@
 
 		&.grouped_form {
 			width: fit-content;
+			word-break: normal;
```

Comment:
This will prevent every text under the grouped product form from being shown oddly.